ports/mecab/portfile.cmake

```
@@ -0,0 +1,66 @@
+# Common Ambient Variables:
+#   CURRENT_BUILDTREES_DIR    = ${VCPKG_ROOT_DIR}\buildtrees\${PORT}
+#   CURRENT_PACKAGES_DIR      = ${VCPKG_ROOT_DIR}\packages\${PORT}_${TARGET_TRIPLET}
+#   CURRENT_PORT_DIR          = ${VCPKG_ROOT_DIR}\ports\${PORT}
+#   PORT                      = current port name (zlib, etc)
+#   TARGET_TRIPLET            = current triplet (x86-windows, x64-windows-static, etc)
+#   VCPKG_CRT_LINKAGE         = C runtime linkage type (static, dynamic)
+#   VCPKG_LIBRARY_LINKAGE     = target library linkage type (static, dynamic)
+#   VCPKG_ROOT_DIR            = <C:\path\to\current\vcpkg>
+#   VCPKG_TARGET_ARCHITECTURE = target architecture (x64, x86, arm)
+#
+

```

I'm not sure, but can you add some codes like this to check `UWP`, `ARM` and `ARM64`?
```suggestion
if (UWP OR ARM OR ARM64)
    message(FATAL_ERROR "mecab does not support on this platform")
```

---

`VCPKG_TARGET_ARCHITECTURE` and `VCPKG_TARGET_IS_UWP` would work.

---

added related code